nomad/grapl-core.nomad

```diff
@@ -40,60 +40,78 @@ variable "dgraph_shards" {
     default = 1
 }
 
-# Where is redis located?
 variable "redis_endpoint" {
     type = string
+    description = "Where can services find redis?"
 }
 
-# What is the name of the schema table?
 variable "schema_table_name" {
     type = string
+    description = "What is the name of the schema table?"
 }
 
 # https://github.com/grapl-security/grapl/blob/af6f2c197d52e9941047aab813c30d2cbfd54523/pulumi/infra/dynamodb.py#L118
-# Without deployment name
 variable "session_table" {
     type = string
     default = "dynamic_session_table"
 }
 
-# How many graph mergers should be running in tandem
 variable "graph_mergers" {
```

Comment:
I'd personally prefer `num_graph_merger` et al but, certainly not a blocking suggestion

Reply:
in fact: this is just busywork for you, i don't care, please ignore

Reply:
Counterpoint: clarity in naming is important, because we're going to be reading this code a lot more than we're writing it.